test/optim/test_optim.py

```diff
@@ -2513,10 +2524,10 @@ def test_radam(self):
 
     @unittest.skipIf(not TEST_CUDA, "test requires CUDA")
     def test_defaults_changed_to_foreach(self):
-        from torch.optim import (adam, adamw, nadam, sgd, radam, rmsprop, rprop,
+        from torch.optim import (adam, nadam, sgd, radam, rmsprop, rprop,
                                  asgd, adamax, adadelta, adagrad)
         multi_optims = ((Adam, adam, "_multi_tensor_adam"),
-                        (AdamW, adamw, "_multi_tensor_adamw"),
+                        (AdamW, adam, "_multi_tensor_adam"),
```

Comment:
```suggestion
                        (AdamW, adam, "_multi_tensor_adam"),  # adamw dispatches to superclass's adam
```
just for clearer docs for future readers